### PR TITLE
Blogpost Students 2016

### PR DESCRIPTION
**Pull request by mkalininait**

### CODE REVIEW

File: blog/_posts/2016-08-29-students-2016.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <a href="/blog/2016-07-21-start-of-loadtocode" title="Team LoadToCode" class="row4 col4"></a>
+
+    <a href="/blog/2016-08-17-Introducing-Team-Fedex++" title="Team Fedex++" class="row5 col1"></a>
+    <a href="/blog/2016-07-14-team-joda-says-hello" title="Team joda" class="row5 col2"></a>
+    <a href="/blog/2016-08-05-B-More-Stunners-puts-Hello-World" title="Team B'More Stunners" class="row5 col3"></a>
+    <a href="/blog/2016-07-18-team-mitpal-getting-it-started" title="Team MitPal" class="row5 col4"></a>
+
+  </div>
+</section>
+
```
Comment (alicetragedy) [inline blog/_posts/2016-08-29-students-2016.md:52]: please add image credits like we have in the other posts

**Comment by alicetragedy** [comment]
hey @mkalininait, the blog post looks great (!!!) and almost ready to merge! Like it says in the [contribution guide](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images), all images need to have an image credit under them. It might also be good to compress the image a bit to reduce its size, because it's bigger than 1MB, which is a lot for everyone cloning our repo. Can you do that? We can publish it right after.

**Comment by mkalininait** [comment]
@alicetragedy I've compressed the image and added credits, please take a look. If the credits are wrong, I'll fix them.

**Comment by alicetragedy** [comment]
Awesome! 👍

**Pull request merged**